arches/app/views/resource.py

```diff
@@ -217,19 +209,34 @@ def get(
                 if append_tile is True:
                     provisionaltiles.append(tile)
             tiles = provisionaltiles
-        map_layers = models.MapLayer.objects.all()
-        map_markers = models.MapMarker.objects.all()
-        map_sources = models.MapSource.objects.all()
-        geocoding_providers = models.Geocoder.objects.all()
+
+        serialized_graph = None
+        if graph.publication and graph.publication.serialized_graph:
+            serialized_graph = graph.publication.serialized_graph
+
+        if serialized_graph:
+            serialized_cards = serialized_graph["cards"]
+            cardwidgets = [
+                widget
+                for widget in models.CardXNodeXWidget.objects.filter(
```

Comment:
I think that here again we can avoid an extra query to the database if we just create a list of models from the serialized dicts (as in graph.py)